package.json

```diff
@@ -149,7 +149,7 @@
         "alex": "^11.0.1",
         "cem-plugin-module-file-extensions": "^0.0.5",
         "chromatic": "^11.20.0",
-        "chromedriver": "^134.0.5",
+        "chromedriver": "^136.0.0",
```

Comment:
This seems unrelated to the dependency work outlined in the description. Can we pull this out since its impact ends up being larger than the card component?

Reply:
Funny! I think @blunteshwar  accidentally pushed it to my branch :P